load-grafana.sh

```diff
@@ -49,7 +52,11 @@ for f in scylla-dash scylla-dash-per-server scylla-dash-io-per-server scylla-das
     if [ -e grafana/$f.$v.template.json ]
     then
         if [ ! -f "grafana/build/$f.$v.json" ] || [ "grafana/build/$f.$v.json" -ot "grafana/$f.$v.template.json" ]; then
-            ./make_dashboards.py -t grafana/types.json -d grafana/$f.$v.template.json
+	    if [ "$KUBERNETES_OPERATOR_BUILD" = true ]; then
```

Comment:
no need for the extra if, just set KUBERNETES_OPERATOR_BUILD to --kubernetes-operator

Reply:
ok